src/components/EscDshotDirection/EscDshotDirectionMotorDriver.js

```diff
@@ -122,6 +123,23 @@ class EscDshotDirectionMotorDriver
         buffer.push8(direction);
         buffer.push8(DshotCommand.dshotCommands_e.DSHOT_CMD_SAVE_SETTINGS);
         this._EscDshotCommandQueue.pushCommand(MSPCodes.MSP2_SEND_DSHOT_COMMAND, buffer);
+
+        let logString = "";
+        if (motorIndex === DshotCommand.ALL_MOTORS) {
+            if (direction === DshotCommand.dshotCommands_e.DSHOT_CMD_SPIN_DIRECTION_1) {
+                logString += i18n.getMessage('motorsText').concat(': ', i18n.getMessage('escDshotDirectionDialog-Open')).concat(' ',i18n.getMessage('escDshotDirectionDialog-CommandNormal'));
+            } else {
+                logString += i18n.getMessage('motorsText').concat(': ', i18n.getMessage('escDshotDirectionDialog-Open')).concat(' ',i18n.getMessage('escDshotDirectionDialog-CommandReverse'));
+            }
+        } else {
+            const  motorNumber = motorIndex+1;
+            if (direction === DshotCommand.dshotCommands_e.DSHOT_CMD_SPIN_DIRECTION_1) {
+                logString += i18n.getMessage(`motorNumber${motorNumber}`).concat(': ', i18n.getMessage('escDshotDirectionDialog-Open')).concat(' ',i18n.getMessage('escDshotDirectionDialog-CommandNormal'));
+            } else {
+                logString += i18n.getMessage(`motorNumber${motorNumber}`).concat(': ', i18n.getMessage('escDshotDirectionDialog-Open')).concat(' ',i18n.getMessage('escDshotDirectionDialog-CommandReverse'));
+            }
+        }
```

Comment:
I'm a bit confused on what's happening here with all the `+=`

From what I understand all of these states are mutually exclusive aka only one of them can exist at a single point in time. `+=` means that we are appending to the existing string.

Wouldn't it just work with simple assignment `=`?

Then another comment is about usage of `concat`, maybe string literals would be a better choice?

So things like:
```
logString += i18n.getMessage(`motorNumber${motorNumber}`).concat(': ', i18n.getMessage('escDshotDirectionDialog-Open')).concat(' ',i18n.getMessage('escDshotDirectionDialog-CommandNormal'));
```
becomes
```
logString = `${i18n.getMessage(`motorNumber${motorNumber}`)}: ${i18n.getMessage('escDshotDirectionDialog-Open')} ${i18n.getMessage('escDshotDirectionDialog-CommandNormal')}`;
```

Reply:
yes, i wanted use `+=` logic, and upon success of prototyping, i forgot to continue such direction.  my next commit solves this with "proper" `+=` usage.